### ISSUE TITLE
sp.corpusplayer buggy when scheduler in overdrive

### ISSUE THREAD

**Issue by jamesstaub** [rconstanzo/SP-tools#1]
Thanks for making this amazing library. I am experiencing some strange behavior with sp.corpusplayer where after a while it stops responding to messages. I am sending the player a list of slice, polybuffer_id (from corpus match), descriptorframe, melbandframe

If I toggle audio on/off it begins working again. 

This video demonstrates the behavior where banging the message with a metro does not work, however clicking the message with a mouse always works. If I toggle audio then the bangs from the metro work for a bit, but eventually stop.

This only seems to happen when scheduler is in overdrive mode

https://user-images.githubusercontent.com/1205113/200140714-630db601-f815-4fe6-b1e7-52c28414c2f2.mov

**Comment by rconstanzo** [comment]
Hmm, testing now, it works fine for me with overdrive on or off, but it does behave a bit weird with audio interrupt turned on, which is strange as it works fine in M4L and as far as I know it's enabled in there by default.

I presume you are on the latest version?
Does it still happen if you remove the extra buffers? (not that it should matter, to narrow things down).

**Comment by jamesstaub** [comment]
Yep latest version, Mac OSX. I set `@voices 1` and still get the behavior, is that what you mean? Just checked and you're right it is due to Audio Interrupt, not Scheduler Overdrive. In any case my patch works fine without that setting.
